### DISCUSSION TITLE
posts/gcn-basis-graphconv-gatconv-sageconv-implementation-pyg-dgl/

### DISCUSSION THREAD

**Discussion by giscus** [pluveto/blog-comments#2] [Announcements]
# posts/gcn-basis-graphconv-gatconv-sageconv-implementation-pyg-dgl/

https://www.less-bug.com/posts/gcn-basis-graphconv-gatconv-sageconv-implementation-pyg-dgl/

**Comment by Hinataee** [comment]
SAGEConv的PyG代码中 当前节点的特征 与  聚合的邻居节点特征 似乎没有进行拼接QAQ
